### DISCUSSION TITLE
OKD 4 installation

### DISCUSSION THREAD

**Discussion by ghost** [okd-project/okd#1230] [General]

**Comment by vrutkovs** [comment]
Please attach [log bundle](https://docs.okd.io/latest/installing/installing-troubleshooting.html#installation-bootstrap-gather_installing-troubleshooting)

**Reply by vrutkovs** [reply]
>May 22 10:41:22 VTNFECONPLA1 hyperkube[1687]: E0522 10:41:22.105741    1687 kubelet_node_status.go:94] "Unable to register node with API server" err="Post \"https://api-int.okd4test.demo.local:6443/api/v1/nodes\": dial tcp: lookup api-int.okd4test.demo.local: Temporary failure in name resolution" node="vtnfeconpla1"

master node can't resolve `api-int` DNS name

**Reply by vrutkovs** [reply]
You've specified 3 control plane nodes:
```
    controlPlane:
      name: master
      replicas: 3
```
But logs were fetched from just one master - did the rest boot?
